dist/test_data.py

```diff
@@ -244,6 +244,11 @@ def __ge__(self, other):
         The duration that the test run will last''', min=0, max=1000000),
     Config('enable_logging', 'false', r'''
         Enables write ahead logs''', type='boolean'),
+    Config('in_memory', 'false', r'''
+        Enables WiredTige's in memory mode.''', type='boolean'),
+    Config('sweep_interval', 10, r'''
```

Comment:
A->Z

Reply:
What.

Reply:
I think he means alphabetize and move `sweep_interval` below.

Reply:
Oh right I saw in_memory and it was alphabetically sorted already.

Reply:
d3e0b8ef2